src/characters/baseActionStates.js

```diff
@@ -1,8 +1,29 @@
+import {player, cS, drawVfx, screenShake, percentShake, finishGame, palettes, pPal,stage} from "../main/main";
+import {sounds} from "../main/sfx";
```

Comment:
Can you fix paths like these to not be relative? Webpack resolves at the root of src, so you can just do `main/sfx` etc.